docs/standards/news.rst

```diff
@@ -3,22 +3,119 @@
 Schema Release Notes
 ================================================================================
 
-Version 1.4.0-dev: (In development)
+Version 1.4.0:
 --------------------------------------------------------------------------------
 
-**Version 1.4-dev, In development.**
-
-New Schema:
-
-1. Introduced the ``RepertoireGroup`` Schema for describing sets of repertoires
-   to be analyzed together. Has support for defining a time series using ``TimePoint``.
+**Version 1.4 schema release.**
+
+New General Purpose Schema:
+
+1. Introduced the experimental ``DataFile`` object, which defines a JSON file
+   holding Repertoire metadata, data processing analysis objects, or any object
+   in the AIRR Data Model.
+2. Introduced the experimental ``RepertoireGroup`` Schema for describing sets
+   of repertoires to be analyzed together.
+3. Introduced the ``InfoObject`` object, which provides information about
+   data and ADC API responses.
+4. Introduced the ``TimePoint`` Schema for defining the time point at which an
+   observation or other action was performed.
+5. Introduced the ``Acknowledgement`` Schema to define contributors to a study,
```

Comment:
This object was introduces via the germline schema and overlaps in terms of content with several other existing fields (as discussed in #552). Therefore it should be listed as part of the Germline Schema.

Reply:
right, I might suggest even leaving `Acknowledgement` out of the news, just to prevent people from actively using outside of germline until #552 is resolved.

Reply:
Moved to the germline section in 50a9d9b.